great-idea/css/index.css

```diff
@@ -258,4 +258,55 @@ footer {
   list-style-position: inside;
   line-height: 2;
   margin-top: 15px;
-}
\ No newline at end of file
+}
+
+strong {
+  font-weight: 700;
+  font-size: 30px;
+}
+
+
+.vision-header-img {
+	margin-top: 30px;
+}
+
+span {
+  text-decoration: line-through;
+}
+
+
```

Comment:
Great job adding in the media query for mobile only especially since we have not covered this yet